tests/storage/README.md

```diff
@@ -9,18 +9,29 @@ or fail (depending on the test).
 
 To setup storage for these tests, run:
 
-    python tests/storage/userstorage.py setup
+    make storage
 
 This can be run once when setting up development environment, and must
 be run again after rebooting the host.
 
+The storage is configured in the file `storage.py` in the root of the
+project. Tests that use user storage load this configuration file and
+access the storage via the BACKENDS dict.
```

Comment:
This is not true (yet) - this describes how upstream userstorage works (used in imageio) but we still use the old userstorage version (tests/storage/userstorage.py).